apps/src/lib/client/tx.rs

```diff
@@ -861,6 +929,27 @@ async fn process_tx(
     }
 }
 
+async fn account_hash_enought_balance(
+    client: &HttpClient,
+    address: &Address,
+    min_amount: Amount,
+) -> bool {
+    let balance_key = token::balance_key(&m1t(), address);
+    match rpc::query_storage_value::<Amount>(client, &balance_key).await {
+        Some(amount) => {
+            if amount < min_amount {
+                eprintln!("Address {} doesn't have enough funds.", address);
+                safe_exit(1);
+            }
+            true
+        }
+        None => {
+            eprintln!("Can't find address {}.", address);
+            safe_exit(1);
+        }
+    }
+}
+
```

Comment:
```suggestion
async fn get_token_balance(
    client: &HttpClient,
    token: &Address,
    address: &Address,
) -> Option<token::Amount> {
    let balance_key = token::balance_key(token, address);
    rpc::query_storage_value::<token::Amount>(client, &balance_key)
}
```

and then at the caller side, e.g.:
```
        // default == 0
        let balance = get_token_balance(&client, &m1t(), &proposal.author).await.unwrap_or_default();
        if balance < min_amount {
            eprintln!("Address {} doesn't have enough funds.", address);
            safe_exit(1);
        }
        ...
```

Reply:
`get_token_balance` should probably go in the queries module, but we can re-use it in couple more places